br/pkg/task/stream.go

```diff
@@ -41,40 +43,56 @@ import (
 )
 
 const (
-	flagStreamTaskName        = "task-name"
-	flagStreamTaskNameDefault = "all" // used for get status for all of tasks.
-	flagStreamStartTS         = "start-ts"
-	flagStreamEndTS           = "end-ts"
-	flagGCSafePointTTS        = "gc-ttl"
+	flagStreamTaskName          = "task-name"
+	flagStreamTaskNameDefault   = "all" // used for get status for all of tasks.
+	flagStreamStartTS           = "start-ts"
+	flagStreamEndTS             = "end-ts"
+	flagGCSafePointTTS          = "gc-ttl"
+	flagStreamRestoreTS         = "restore-ts"
+	flagStreamFullBackupStorage = "full-backup-storage"
 )
 
 var (
-	StreamStart  = "stream start"
-	StreamStop   = "stream stop"
-	StreamPause  = "stream pause"
-	StreamResume = "stream resume"
-	StreamStatus = "stream status"
+	StreamStart   = "stream start"
+	StreamStop    = "stream stop"
+	StreamPause   = "stream pause"
+	StreamResume  = "stream resume"
+	StreamStatus  = "stream status"
+	StreamRestore = "stream restore"
 )
 
 var StreamCommandMap = map[string]func(c context.Context, g glue.Glue, cmdName string, cfg *StreamConfig) error{
-	StreamStart:  RunStreamStart,
-	StreamStop:   RunStreamStop,
-	StreamPause:  RunStreamPause,
-	StreamResume: RunStreamResume,
-	StreamStatus: RunStreamStatus,
+	StreamStart:   RunStreamStart,
+	StreamStop:    RunStreamStop,
+	StreamPause:   RunStreamPause,
+	StreamResume:  RunStreamResume,
+	StreamStatus:  RunStreamStatus,
+	StreamRestore: RunStreamRestore,
 }
 
 // StreamConfig specifies the configure about backup stream
 type StreamConfig struct {
 	// common part that all of stream commands need
 	Config
-	TaskName string `json:"task-name" toml:"task-name"`
 
-	// startTs usually equals the tso of full-backup, but user can reset it
+	// FullBackupStorage is used to find the maps between table name and table id during restoration.
+	// if not specified. we cannot apply kv directly.
+	FullBackupStorage string `json:"full-backup-storage" toml:"full-backup-storage"`
```

Comment:
Maybe move it to `StreamRestoreConfig`?

Reply:
currently we don't have `StreamRestoreConfig`. 🤔. I can add a new config. 
`StreamRestoreConfig` and `StreamConfig` both need common `Config` but each has different compatible config.
for example. StreamRestoreConfig needs `full-backup-storage` but doesn't need `task-name`.

so maybe we should change `br stream restore` to `br restore stream`. WDYT?